### DISCUSSION TITLE
Issue: failure anomalies - [resource name]

### DISCUSSION THREAD

**Discussion by cramsey86** [PrateekKumarSingh/AzViz#28] [Q&A] [answered]
I am getting Edge and Nodes that are being created with the names "failure anomalies - [resource name]". They have the generic icon, so I am not sure what resource it is actually failing on, and the temporary armtemplate.json is being cleaned up before I have a chance to view where the issue might be.

![image](https://user-images.githubusercontent.com/49802142/114930625-d7a17680-9dfa-11eb-98b4-47c8330a3222.png)

**Comment by PrateekKumarSingh** [comment, marked as answer]
Hey @cramsey86,

Can you try running the command again with `-LabelVerbosity 2` like in the following example and it should give you info like  `resource provider` and `type`

```PowerShell
Export-AzViz -ResourceGroup demo-2, demo-3 -LabelVerbosity 2 -CategoryDepth 1 -Theme light -Verbose -Show -OutputFormat png
```

Other option is simply export the resource group you are targeting to check the ARM template: `Export-AzResourceGroup -ResourceGroupName $Target `
